1.8/usage/tutorials/index.md

```diff
@@ -2,4 +2,39 @@
 post_title: Tutorials
 menu_order: 90 
 ---
-This is a collection of tutorials about using DC/OS. Learn how to run services and operate services in production.
+
+This is the DC/OS tutorials section, covering how to use DC/OS components as well as stateless and stateful DC/OS services. Typically, the services discussed here are available and installable from the [Universe](https://github.com/mesosphere/universe).
+
+The left-hand side menu lists the available basic tutorials, such as how to create and run a service. Further, there are a number of examples of how to use DC/OS services such as Spark or WordPress, which are available via the [dcos/examples](https://github.com/dcos/examples/) GitHub repo.
```

Comment:
Do we want to explain here why we have separate locations for these tutorials (e.g. examples located in GH repo are tested/validated?). In looking what we have now - it just seems like a formality that the content is displayed in one format or another. Also, explaining a left-hand menu seems irrelevant since the links are also repeated in the body.  

CC: @williamsaaron